mix.exs

```diff
@@ -11,18 +11,16 @@ defmodule Traceman.Mixfile do
     ]
   end
 
-  # Run "mix help compile.app" to learn about applications.
   def application do
     [
-      extra_applications: [:logger]
+      extra_applications: [:logger, :cowboy, :plug]
     ]
   end
 
-  # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      # {:dep_from_hexpm, "~> 0.3.0"},
-      # {:dep_from_git, git: "https://github.com/elixir-lang/my_dep.git", tag: "0.1.0"},
+      {:cowboy, "~> 2.0"},
+      {:plug, "~> 1.0"}
```

Comment:
Would be nice if the service could define these, I wonder if there is a pattern for that.

Reply:
I guess. The tests would not work, but I can check if it would compile correctly.

Reply:
Maybe somthing like, `{:cowboy, "~> 2.0", only: test}` would work? (never tried this)

Reply:
It seems to work! 👍 I'll check how it behaves when added as a dependency.